tests/model/test_scanvi.py

```diff
@@ -578,3 +578,63 @@ def check_no_logits_and_softmax(model: SCANVI):
 
     model = SCANVI.load(resave_model_path, adata)
     check_no_logits_and_softmax(model)
+
```

Comment:
Shouldn't this be in another test all together?

Reply:
ok